File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.java
class Solution {
    public List<List<Integer>> findDifference(int[] nums1, int[] nums2) {
        List<List<Integer>> result=new ArrayList<>();
        HashSet<Integer> temp1=new HashSet<>();
        HashSet<Integer> temp2=new HashSet<>();

        for(int i=0; i<nums1.length; i++){
            boolean found = false;
            for(int j=0; j<nums2.length; j++) {
                if(nums1[i] == nums2[j]) {
                    found = true;
                    break;
                }
            }
            if(!found){
                temp1.add(nums1[i]);
            }
        }
        
        for(int i=0; i<nums2.length; i++){
            boolean found = false;
            for(int j=0; j<nums1.length; j++) {
                if(nums2[i] == nums1[j]) {
                    found = true;
                    break;
                }
            }
            if(!found){
                temp2.add(nums2[i]);
            }
        }
        
        List<Integer> temp1List = new ArrayList<>(temp1);
        List<Integer> temp2List = new ArrayList<>(temp2);
        result.add(temp1List);
        result.add(temp2List);
        return result;
    }
}

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution(object):
    def findDifference(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[List[int]]
        """
        result=[]
        temp1=set()
        temp2=set()
        for i in range(len(nums1)):
            if nums1[i] not in nums2:
                temp1.add(nums1[i])
        for i in range(len(nums2)):
            if nums2[i] not in nums1:
                temp2.add(nums2[i])
        result.append(list(temp1))
        result.append(list(temp2))
        return result